{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2020-36242",
      "description": "In the cryptography package before 3.3.2 for Python, certain sequences of update calls to symmetrically encrypt multi-GB values could result in an integer overflow and buffer overflow, as demonstrated by the Fernet class.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2020-36242",
      "reportURL": "https://github.com/pyca/cryptography/issues/5615",
      "timeStamp": "2021-07-21T11:39:00Z"
    }
  ],
  "trickest": [
    {
      "id": "CVE-2020-36242",
      "url": "https://github.com/pyca/cryptography/blob/master/CHANGELOG.rst",
      "added_at": "1970-01-01T00:00:00Z"
    }
  ],
  "nomisec": [],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2020-36242",
      "url": "https://github.com/pyca/cryptography/blob/master/CHANGELOG.rst",
      "added_at": "1970-01-01T00:00:00Z",
      "score": 0,
      "template_for": ""
    },
    {
      "cve": "CVE-2020-36242",
      "url": "https://github.com/pyca/cryptography/issues/5615",
      "added_at": "2021-07-21T11:39:00Z",
      "score": 0.4,
      "template_for": ""
    }
  ]
}
